test/kubemark/start-kubemark.sh

```diff
@@ -338,8 +353,11 @@ function wait-for-hollow-nodes-to-run-or-timeout {
 }
 
 ############################### Main Function ########################################
+detect-project &> /dev/null
```

Comment:
I'm not really convinced that we should have this function as part of kubemark skeleton, because this function is something that should be defined upstream, which is in the cloud-provider's util. Like how we currently have it defined in `cluster/$CLOUD_PROVIDER/util.sh` (for gce, aws and gke).